lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -237,6 +237,21 @@ def instructor_dashboard_2(request, course_id):  # lint-amnesty, pylint: disable
 
     certificate_invalidations = CertificateInvalidation.get_certificate_invalidations(course_key)
 
+    # x block asides enabled, included rapid response tab
+    if 'ol_openedx_rapid_response.app.RapidResponsePluginConfig' in settings.INSTALLED_APPS:
+        from openedx.core.djangoapps.plugins.constants import ProjectType
+        from edx_django_utils.plugins import get_plugins_view_context
+        from ol_openedx_rapid_response.constants import RAPID_RESPONSE_PLUGIN_VIEW_NAME  # pylint: disable=import-error
```

Comment:
These inline imports aren't great for a few reasons. One reason is that if there's an import error, we wouldn't know until this function is called. I tried this locally and it seems to work.

Add this to the bottom of the imports at the top of the module:
```python
rapid_response_enabled = 'ol_openedx_rapid_response.app.RapidResponsePluginConfig' in settings.INSTALLED_APPS
if rapid_response_enabled:
    from openedx.core.djangoapps.plugins.constants import ProjectType
    from edx_django_utils.plugins import get_plugins_view_context
    from ol_openedx_rapid_response.constants import RAPID_RESPONSE_PLUGIN_VIEW_NAME  # pylint: disable=import-error
```

Then update this block:
```suggestion
    if rapid_response_enabled:
```

Reply:
I gave it a more cleaner look instead of the checking the plugin installation and then fetching the context we'll just fetch the context which will be a `dict/None` based on if the plugin is installed or not. Also there has been an [Upstream PR](https://github.com/edx/edx-platform/pull/29376/files#diff-0d1ee66fb84bee6c3517980a7cefd74a050ccb60da9c0c3b662ab0a2d80d17c0R255) for making more general improvements and when that's merged we'll make our plugins context generic too.